airsonic-main/src/main/java/org/airsonic/player/service/metadata/FFmpegParser.java

```diff
@@ -99,6 +101,76 @@ public MetaData getRawMetaData(File file) {
                     break;
                 }
             }
+
+            /*
+             * Find tags, if any
+             * tag names are forced to lower case
+             */
+            final JsonNode tags = result.at("/format/tags");
+            if (tags != null) {
+                Iterator<String> it = tags.fieldNames();
+                String key = null;
+                while (it.hasNext()) {
+                    key = it.next();
+                    // skip empty tags
+                    if (StringUtils.isBlank(tags.get(key).asText())) {
```

Comment:
`tags.get(key).asText()` should be stored in a local variable instead of being repeated everywhere

Reply:
That is certainly possible but it doesn't make things faster as it only gets called once per switch cycle and it takes an extra allocation of a local. Anyway, will do. The same can be done in Jaudiotagger (f.toString())

Reply:
It' more about simplifying/reducing the amount of code than about performances :)